tests/load/locustfile.py

```diff
@@ -104,7 +108,9 @@ def register(self) -> None:
         with self._time_event(name="register"):
             body = json.dumps(dict(messageType="register", channelID=chid))
             self.ws.send(body)
-            res = json.loads(self.ws.recv())
+            reply = self.ws.recv()
+            assert reply, "No 'register' response"
+            res = json.loads(reply)
```

Comment:
Wouldn't ```json.loads``` fail if the variable ```reply``` was empty?

Reply:
My understanding is if `reply` is empty the assertion will fail, raising an exception, and the code on line 113 should not be executed.

Reply:
Yeah, sorry if it wasn't very explicit.

My hope is that since we can't examine the logs easily, we could use the assert errors to diagnose which of the messages are coming back as empty. These won't explain why those messages are coming back empty, but there are some assumptions we can make.

If the `hello` is coming back empty, then there was something wrong with the sent `hello` (conflicting UAID, bad value, timeout, etc.) and the server terminated the connection, which resulted in the empty packet being sent.

If the `register` came back empty, then there was a problem with the register command (e.g. the requested ChannelID was not lower case & hyphenated,  or the endpoint generation failed, etc.)

Reply:
@jrconlin should I add some additional information to the assertion, perhaps print the channelID?